apps/camly_app/ios/scripts/retrieve_dart_defines.sh

```
@@ -0,0 +1,25 @@
+# Dart-defineを書き込んだり、Flavorごとのxcconfigをincludeするファイル
+OUTPUT_FILE="${SRCROOT}/Flutter/DartDefines.xcconfig"
+
+# Flutter 2.2 以降で必要な、Dart-Definesのデコード処理
+function decode_url() { echo "${*}" | base64 --decode; }
+
+# 最初にファイル内容をいったん空にする
+: > $OUTPUT_FILE
+
+IFS=',' read -r -a define_items <<<"$DART_DEFINES"
+
+for index in "${!define_items[@]}"
```

Shellcheck

[warning] In POSIX sh, indirect expansion is undefined.

(SC3053)